internal/task/task/bs/format.go

```diff
@@ -137,12 +138,14 @@ func (s *step2EditFSTab) execute(ctx *context.Context) error {
 			ExecOptions: curveadm.ExecOptions(),
 		})
 	}
-	steps = append(steps, &step.Sed{ // add new record
-		Files:       []string{os.GetFSTabPath()},
-		Expression:  &express2add,
-		InPlace:     true,
-		ExecOptions: curveadm.ExecOptions(),
-	})
+	if s.skipAdd {
```

Comment:
Wrong judge, i think you want to express is that the step will not add new record if the parameter `skipAdd` is true.

Reply:
yeh ,Thanks you review it
we have two curveadm project for backup
and test is <strong> deleteFSTab<strong> ,so I did not find the error
I've resolve it and retested uploading tested pass images